embed/script.js

```diff
@@ -0,0 +1,252 @@
+function h() {
```

Comment:
A more descriptive function name would be good - as it's not obvious from the outset

Reply:
Vahiguru Jee. `h` is actually the name of this lib, inspired from [hyperscript](https://github.com/hyperhype/hyperscript). We used to use this when our codebase was in jQuery, and sttm-desktop still uses it. So the devs are versed with its functionality. Either way, we can refactor this script anytime in future.